dev/bootstrap.cfg

```diff
@@ -10,6 +10,7 @@ puppetlabs.services.legacy-routes.legacy-routes-service/legacy-routes-service
 puppetlabs.services.puppet-admin.puppet-admin-service/puppet-admin-service
 puppetlabs.trapperkeeper.services.authorization.authorization-service/authorization-service
 puppetlabs.services.versioned-code-service.versioned-code-service/versioned-code-service
+puppetlabs.trapperkeeper.services.scheduler.scheduler-service/scheduler-service
```

Comment:
Also a question for testing this in the repl...I've got this change, but it doesn't look like it was properly loaded when I attempt to start puppetserver through the repl (it works if I do `lein run -c ~/.puppetserver/puppetserver.conf` though)

```
user=> (go)
2016-07-19 15:56:17,562 DEBUG [nREPL-worker-0] [p.t.internal] Initializing lifecycle worker loop.

RuntimeException Service ':SchedulerService' not found  puppetlabs.trapperkeeper.internal/handle-prismatic-exception! (internal.clj:98)
```